frontend/src/components/common/GenericModal.tsx

```
@@ -57,7 +57,7 @@ const GenericModal = (props: ModalProps) => {
 
   return (
     <Container>
-      <Modal show={show} onHide={ok}>
+      <Modal show={show} onHide={close}>
```

the default when the modal hides is whatever is configured for cancel.